QA/government_interest/GovtInterestTester.py

```diff
@@ -117,11 +117,17 @@ def generate_govt_int_samples(self):
          ON pca.patent_id = gi.patent_id
        LEFT JOIN `patent_govintorg` pg
          ON pg.patent_id = gi.patent_id
-       LEFT JOIN government_organization go
+       LEFT JOIN patent.government_organization go
```

Comment:
@bethanne-card  Can you switch this to use the database name from the config ? The key I believe is "RAW_DB"